ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcarousel/v1/productcarousel/_cq_dialog/.content.xml

```diff
@@ -3,25 +3,87 @@
     xmlns:cq="http://www.day.com/jcr/cq/1.0"
     xmlns:jcr="http://www.jcp.org/jcr/1.0"
     xmlns:nt="http://www.jcp.org/jcr/nt/1.0" 
+    xmlns:granite="http://www.adobe.com/jcr/granite/1.0" 
     jcr:primaryType="nt:unstructured" 
     jcr:title="Product Carousel"
-    extraClientlibs="[cif.shell.picker]"
+    extraClientlibs="[core.cif.components.productcarousel.v1.editor,cif.shell.picker]"
     trackingFeature="cif-core-components:productcarousel:v1">
-    <content jcr:primaryType="nt:unstructured">
+    <content jcr:primaryType="nt:unstructured"
+        granite:class="cif-product-carousel__editor">
         <items jcr:primaryType="nt:unstructured">
             <column jcr:primaryType="nt:unstructured">
                 <items jcr:primaryType="nt:unstructured">
-                    <content jcr:primaryType="nt:unstructured"
-                        sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                    <selectionType jcr:primaryType="nt:unstructured"
                         sling:orderBefore="id"
-                        fieldLabel="Carousel content" 
-                        typeHint="[product@String[]]">
-                        <field jcr:primaryType="nt:unstructured" 
-                            sling:resourceType="commerce/gui/components/common/cifproductfield" 
-                            fieldLabel="Product" 
-                            name="./product" 
-                            selectionId="combinedSku" />
-                    </content>
+                        sling:resourceType="granite/ui/components/coral/foundation/form/radiogroup"
+                        fieldDescription="Select the products one by one or select a category for the products to be displayed."
+                        name="./selectionType"
+                        deleteHint="{Boolean}false"
+                        text="Selection Type"
```

Comment:
IMO the description here is quite technical and not very intuitive. I would change this to:

Label: `Display Products from`
Option product: `Manual Selection`
Option category: `Category`